tasks/eth/1-security-council-phase-0/README.md

```diff
@@ -83,23 +83,37 @@ validate integrity of the simulation, we need to check the following:
 
 Now click on the "State" tab. Verify that:
 
-1. There is only a single state override at address
-   `0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A`, which overrides
-   storage slot `0x4` to new value `0x1`. This override is only
-   intended to change the Foundation multisig's quorum threshold to 1
-   so we can perform a tenderly simulation of the execution,
+1. There are only two single state override at address
+   `0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A`:
+
+   a. One of them overrides storage slot `0x4` to new value
+   `0x1`. This override is only intended to change the Foundation
+   multisig's quorum threshold to 1 so we can perform a tenderly
+   simulation of the execution,
+
+   b. One of them overrides storage slot `0x5` to `0x56`. This
+   override is an no-op to override the nonce of the multisig to 86,
+   which is the same as it's current value.
```

Comment:
Why does this no-op override exist? I'll also note that I've confirmed this is a no-op by running `cast call 0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A "nonce()(uint256)" --rpc-url $MAINNET_RPC_URL`, which might be nice to add here